plugin/src/init.server.lua

```diff
@@ -1,222 +1,289 @@
 --!strict
 local HttpService = game:GetService("HttpService")
-assert(plugin, "This code must run inside of a plugin")
+local RunService = game:GetService("RunService")
+local ScriptEditorService = game:GetService("ScriptEditorService")
+local TestService = game:GetService("TestService")
 
-if game:GetService("RunService"):IsRunning() then
+local IsRunning = RunService:IsRunning()
```

Comment:
I don't really understand the point in putting this into it's own variable, the original seemed fine